CSharpOOP2/CSharpOOP2/Horse.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP2
+{
+    public class Horse
+    {
+        public string Name;
+        public string Color;
+        public double Age;
+        public string Breed;
+        public double TailLength;
+        public bool IsVaccinated;
+        public int DaysNumberLastVaccination;
+        public int _speed;
+        public bool _isReadyToCompetition;
+        public static string _kind;
+        public int horseNumber;
+
+        //TASK_3
+        public bool NeedDoVactination(int DaysNumberLastVactination)
+        {
+            if (DaysNumberLastVactination >= 0 && DaysNumberLastVactination <= 365)
+            {
+                return IsVaccinated = true;
+            }
+            else
+            {
+                return IsVaccinated = false;
+            }
+        }
+
+        //TASK_6
+        public Horse(string name, string color, double age, string breed, double tailLength, bool isVaccinated, int daysNumberLastVaccination)
+        {
+            Name = name;
+            Color = color;
+            Age = age;
+            Breed = breed;
+            TailLength = tailLength;
+            IsVaccinated = isVaccinated;
+            DaysNumberLastVaccination = daysNumberLastVaccination;
+        }
+
+        //TASK_7
+
+        public Horse()
+        {
+        }
+
+        //TASK_8
+        public Horse(string name, string color)
+        {
+            Name = name;
+            Color = color;
+        }
+
+        //TASK_10
+        public void Status(bool isVactinated)

```

Comment:
suggestion: as this is method you have to name it like Do_something, because method is an action :) Example for your code is PrintStatus :)